lambda-layers/package.json

```diff
@@ -31,8 +31,5 @@
   "devDependencies": {
     "@types/node": "18.11.19",
     "typescript": "~5.1.6"
-  },
-  "dependencies": {
```

Comment:
We don't need any Lambda SDK or anything for this package?  Just double checking we tested building and publishing as per https://github.com/aws/aws-rfdk/blob/mainline/lambda-layers/README.md

Reply:
Good catch, thanks!

I updated `lambda-layers` to work with the new v3 SDK, and I was able to run the steps in that README without any errors.